manifests/params.pp

```diff
@@ -15,6 +15,7 @@
   $receive_facts  = true
   # should foreman manage host provisioning as well
   $unattended     = true
+  $unattended_url = "http://${lower_fqdn}:8000"
```

Comment:
This is a non-default location I believe. When is it needed to override this and would it make sense to default to ```undef``` and only place it in the settings when the user overrides it?

Reply:
I can't check right now, but I'm pretty sure that's the current default location.

But sure, I was in two minds of setting it to a default location, or not at all...

Reply:
When would I use it?
Automated deployments of foreman puppet, my current need is to stand everything up from scratch with no user intervention

Reply:
I don't recall having to ever override this and I did do bare metal deployments. Now that might just be my memory but I think by default the unattended url is based on the regular foreman url on port 80. A plain foreman doesn't even listen on port 8000 I think.